scripts/us_census/acs5yr/subject_tables/common/generate_col_map.py

```diff
@@ -232,6 +232,13 @@ def _generate_stat_vars_from_spec(self):
         # Should that be considered an error for subject tables?
         return self.column_map
 
+    def check_obs_props(self, obs_props, stat_var):
+        """Checks for property values match in obs_props"""
+        # Check for measuredProperty
+        if obs_props['mprop'] != stat_var['measuredProperty']:
```

Comment:
what happens if 'mprop' is not present on dict obs_props or 'measuredProperty' is not present in stat_var or both are empty?

perhaps?
if obs_prop.get('mprop', 'mporp') != stat_var.get('measuredProperty', 'measuredProperty'):

Reply:
Thanks for pointing out this case!
I have pushed a fix.